packages/design-tokens/src/getters.ts

```diff
@@ -74,7 +74,6 @@ export function getMediaQuery(options: {
     if (breakpointGTE && !breakpointLT) {
       return css`
         @media ${media} (min-width: ${breakpointGTE}) {
-          font-size: ${baseFontSize};
```

Comment:
I think there are actually some tests asserting for this that need updating.

It reminds me actually that they need to be run as part of CI. I'll add a note to the board.